setup.py

```diff
@@ -89,33 +86,33 @@
                       'tables',
                       'matplotlib<2',
                       'subprocess32'
-                      #pyicu - seems to work without
-                      #pyqt4 - required external program
-                      #PySide - required external program
+                      # pyicu - seems to work without
```

Comment:
@dthomas1953 I'm now noticing that these were removed but there are still checks for PySide and PyQt4.

See: https://github.com/PIC-IRIS/PH5/blob/1bb585aa91ea7ebbea308c2879484af724cab064/setup.py#L28-L48

I think these should be removed to if they are no longer necessary. Also how do the GUI programs work without PyQt? Do they have their own checks for these so that they are not necessary for command line users?

Reply:
We don't need to undo the merge, but I restored the `applinks` branch so this can be fixed.

Reply:
I tried removing the try/except clauses for PyQt4 and PySide, and including them in install_requires, but it does not compile. Seems to me they are still necessary.

Reply:
I'll push a change momentarily for you to test

Reply:
@dthomas1953 Ready for your review. Please see https://github.com/PIC-IRIS/PH5/pull/337